shared/toks-components/src/components/Modal/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { theme } from '@depromeet/theme';
+import styled from '@emotion/styled';
+import { Flex } from '@toss/emotion-utils';
+import React, { Dispatch, ReactNode, SetStateAction } from 'react';
+
+import { Text } from '@depromeet/toks-components';
+
+interface Props {
+  modalTitle: string;
+  contents: ReactNode;
+  button: ReactNode;
+  setIsModalOpened: Dispatch<SetStateAction<boolean>>;
+}
+export function Modal({ modalTitle, contents, button, setIsModalOpened }: Props) {
+  const onClickOutsideModal = () => {
+    setIsModalOpened(false);
+  };
+  const onClickInsideModal = (e: React.MouseEvent) => {
+    e.stopPropagation();
+  };
+  return (
+    <ModalWrapper onClick={onClickOutsideModal}>
+      <ModalContainer onClick={onClickInsideModal}>
+        <Text variant="title03">{modalTitle}</Text>
+        <Flex direction="column">
+          {contents}
+          {button}
+        </Flex>
```

Comment:
요건 children으로 받는게 좋아보입니다~ 정해진 Dialog 모습이 아니라 추상화를 최대한 하는게 좋아보여요
```suggestion
      {children}
```

Reply:
넵 이부분도 고민했는데 그냥 추상화해서 받겠습니당

Reply:
[c14e0de](https://github.com/depromeet/toks-web/pull/122/commits/c14e0de546b3963c9314eb0b827c20637559e947) 추상화하는 방식으로 변경했습니다 !